frontend/src/features/projects/dispose/steps/SelectProjectPropertiesStep.tsx

```diff
@@ -68,19 +84,66 @@ const SelectProjectPropertiesStep = ({ isReadOnly, formikRef }: IStepProps) => {
     [setFilter, setPageIndex],
   );
 
+  // Update filter to only show Surplus Active when clicked
+  const handleShowSurplusClick = () => {
+    setFilter({ ...filter, classificationId: Classifications.SurplusActive.toString() });
```

Comment:
Is it just surplus active or both surplus encumbered?

Reply:
The story said Surplus Active but I was confused by the wording of Surplus Only.. going to confirm with Bobbi!

Reply:
Now changing wording  `Surplus Only` -> `Surplus Active` for the text